src/main/java/com/example/gradecalculator/model/SubjectTO.java

```diff
@@ -0,0 +1,40 @@
+package com.example.gradecalculator.model;
+
+import com.example.gradecalculator.entities.Subject;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+public class SubjectTO {
+
+
+    @Getter
+    @Setter
+    private int id;
+
+    @Getter
+    @Setter
+    private String name;
+
+    @Getter
+    @Setter
+    private String description;
+    @Setter
+    @Getter
+    private Subject subject;
+    @Setter
+    @Getter
+    private boolean selected;
+    @Getter
+    @Setter
+    private String userSubject;
+
+    private List<SubjectTO> subjects;
```

Comment:
would this not also need setters and getters?
also when every field is annotated with setter and getter attribute, just set the attribute to the class instead of fields